apps/integration/src/app/integrations/modal-wait/modal-wait.component.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  inject,
+} from '@angular/core';
+import { SkyModalService } from '@skyux/modals';
+
+import { ModalWaitModalComponent } from './modal-wait-modal.component';
+
+@Component({
+  selector: 'app-modal-wait',
+  templateUrl: './modal-wait.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ModalWaitComponent {
+  public isWaiting = false;
+
+  #changeDetector = inject(ChangeDetectorRef);
+  #modalService: SkyModalService;
+
+  constructor(modalService: SkyModalService) {
+    this.#modalService = modalService;
+  }
+
+  public openModal(): void {
+    this.#modalService.open(ModalWaitModalComponent);
+  }
+
+  public triggerWait(): void {
+    this.isWaiting = true;
+    setTimeout(
+      /* istanbul ignore next */ () => {
+        this.isWaiting = false;
```

Comment:
Does this end up doing anything? I don't see any tests running in `fakeAsync`, which makes me think this might try to run after test is complete and the component is destroyed.

Reply:
I had it there for local development as you can serve this page. I had it there so that the wait would turn back off after 5 seconds. I removed it though - anyone testing locally could do something similar to get it to turn off.